src/api/ens.js

```diff
@@ -4,9 +4,22 @@ import { hash, normalize } from 'eth-ens-namehash'
 import { abi as ensContract } from '@ensdomains/ens/build/contracts/ENS.json'
 import { abi as reverseRegistrarContract } from '@ensdomains/ens/build/contracts/ReverseRegistrar.json'
 import { abi as resolverContract } from '@ensdomains/resolver/build/contracts/PublicResolver.json'
+import { abi as oldResolverContract } from '@ensdomains/ens-022/build/contracts/PublicResolver.json'
 import { abi as fifsRegistrarContract } from '@ensdomains/ens/build/contracts/FIFSRegistrar.json'
 import { abi as testRegistrarContract } from '@ensdomains/ens/build/contracts/TestRegistrar.json'
 
+oldResolverContract.forEach((old, i) => {
```

Comment:
This looks like old piece of code not used anywhere.

Reply:
I just added this back in cos i was getting errors